src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -511,6 +511,23 @@ namespace Sapphire::Network::Packets::Server
     /* 0018 */ uint64_t targetId;
   };
 
+  /**
+  * Structural representation of the packet sent by the server
+  * to update certain player details / status
+  */
+  struct FFXIVIpcActorControl20F : FFXIVIpcBasePacket< ActorControl20F >
+  {
+    /* 0000 */ uint16_t category;
+    /* 0002 */ uint16_t padding;
+    /* 0004 */ Common::DesynthPacketType resultType;
```

Comment:
is this actually logmessage or something?

Reply:
It seems like it's used for both log message and the dialog box that appears to show the item you just desynthed and the result items. I am going to check later today if it's used for reduction result box too, as there's some unused fields that could just be related to reduction (collectability for instance)